packages/ra-ui-materialui/src/list/datagrid/ExpandAllButton.tsx

```diff
@@ -2,19 +2,20 @@ import * as React from 'react';
 import { memo } from 'react';
 import IconButton from '@mui/material/IconButton';
 import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
-import { useTranslate, useExpandAll } from 'ra-core';
+import { useTranslate, useExpandAll, useResourceContext } from 'ra-core';
 import { DatagridClasses } from './useDatagridStyles';
 import clsx from 'clsx';
 
 interface ExpandAllButtonProps {
-    resource: string;
```

Comment:
same

Reply:
Actually this is an internal component only used by our own DatagridHeader, so this doesn't count as a BC